site/content/en/docs/Guides/access-api.md

```diff
@@ -134,6 +134,133 @@ kubectl logs create-gs-6wz86-7qsm5  --namespace agones-system
 ```
 You have just created a GameServer using Kubernetes Go Client.
 
+## Best Practice: Using Informers and Listers
+
+Almost all  Kubernetes' controllers and custom controllers utilise `Informers` and `Listers`
+to reduce the load on the Kubernetes's control plane.
+
+Repetitive, direct access of the Kubernetes control plane API can significantly
+reduce the performance of the cluster -- and Informers and Listers help resolving that issue.
+
+Informers and Listers reduce the load on the Kubernetes control plane
+by creating, using and maintaining and eventually consistent an in-memory-cache.
+This can be watched and also queried with zero cost, since it will only read against
+its in-memory model of the Kubernetes resources.
+
+Informer's role and Lister's role are different.
+The Informer is the mechanism for watching a Kubernetes object's event.
+
+When the Kubernetes objects changes(e.g. CREATE,UPDATE,DELETE), the Informer is informed,
+and can execute a callback with the relevant object as an argument.
+
+This can be very useful for building event based systems against the Kubernetes API.
+
+The Lister is the mechanism for querying Kubernetes object's against the client side in-memory cache.
+Users can list the Kubernetes objects by using Kubernetes APIs, however,
+if users do so, it places load on the Kubernetes control plane.
+
+Since an Lister stores objects in an in-memory cache, queries against a come at practically no cost.
```

Comment:
```suggestion
Listers are a mechanism for querying Kubernetes object's against the client side in-memory cache.
Since an Lister stores objects in an in-memory cache, queries against a come at practically no cost.
```

I don't think we need to repeat the issue with direct acess.

Reply:
Changed from `Listers are a mechanism` to `A Lister is the mechanism` according to `An Informer is the mechanism`.